### DISCUSSION TITLE
Widgets (AcroForms) Button Field Images support?

### DISCUSSION THREAD

**Discussion by aashishvanand** [pymupdf/PyMuPDF#1164] [Looking for help]
Hi,
is there a possibility to add a custom image to the button field? I am looking for some similar implementations such as the one done [here](https://pdf-lib.js.org/#fill-form) (PDF-Lib.js) 
PDF-Lib is a JS library that has support for form fields. In the above example, they create e button field and add Mario image as to that button field

**Comment by JorjMcKie** [comment]
> is there a possibility to add a custom image to the button field? 

Currently there is not. Let me check this out in the next days, but I cannot promise anything.
